tests/serverApi/serverApi-serialization_error-002.phpt

```diff
@@ -6,15 +6,15 @@ MongoDB\Driver\ServerApi unserialization errors (__serialize and __unserialize)
 require_once __DIR__ . '/../utils/basic.inc';
 
 echo throws(function() {
-    unserialize('C:24:"MongoDB\Driver\ServerApi":24:{a:1:{s:7:"version";i:0;}}');
```

Comment:
Ah, I was wondering if we had any examples that still relied on `Serializable::unserialize()`. Missed this earlier, but it's what I had in mind when writing about the possible loss in functionality in https://github.com/mongodb/mongo-php-driver/pull/1663#discussion_r1765303021.